contrib/minizip/unzip.c

```diff
@@ -112,7 +110,7 @@
 # define ALLOC(size) (malloc(size))
 #endif
 #ifndef TRYFREE
-# define TRYFREE(p) { free(p);}
+# define TRYFREE(p) { free(p); }
```

Comment:
Note that the definitions of `TRYFREE` in `unzip.c` and `zip.c` are inconsistent.